flump/__init__.py

```diff
@@ -47,12 +47,10 @@ def do_logging():
 
             debug_string = (
                 "%s request made for resource type %s with kwargs: %s "
-                "and json: %s"
+                "and data: %s"
             )
-
             logger.debug(
-                debug_string, request.method, request.view_args,
-                get_json() if request.method in ('PATCH', 'POST') else ''
+                debug_string, request.method, request.view_args, request.data
```

Comment:
data
> Contains the incoming request data as string in case it came with a mimetype Flask does not handle.

Which means it'll be empty at least some of the time.  Should we try and do something different?